example/redisfailover/pmem.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: storage.spotahome.com/v1alpha2
+kind: RedisFailover
+metadata:
+  name: redisfailover-pmem
+spec:
+  sentinel:
+    replicas: 3
+    command:
+      - "redis-server"
+      - "/redis/sentinel.conf"
+      - "--sentinel"
+      - "--protected-mode"
+      - "no"
+  redis:
+    replicas: 3
+    image: redis_pmem	# From https://github.com/pmem/pmem-redis (needs to be built and uploaded to a repository to be used)
```

Comment:
Is there a real image (uploaded to a registry), that can be set as an example?

Reply:
At the moment, the image is unavailable as mentioned in the comment. 
For testing purposes I've used one from an internal registry.
Should I add a _example requirements_ documentation in order to use this example at docs folder?